packages/office-ui-fabric-react/src/components/DocumentCard/examples/DocumentCard.Basic.Example.tsx

```diff
@@ -8,9 +8,8 @@ import {
 } from 'office-ui-fabric-react/lib/DocumentCard';
 import { ImageFit } from 'office-ui-fabric-react/lib/Image';
 import { TestImages } from '../../../common/TestImages';
-import './DocumentCard.Example.scss';
 
-export class DocumentCardBasicExample extends React.Component<any, any> {
+export class DocumentCardBasicExample extends React.PureComponent<{}> {
```

Comment:
nit: [`<{}>` is redundant here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L471).

I prefer to avoid redundant type arguments but it's your call. I can see an argument for being extra clear to readers that there are no props.

Reply:
Similarly elsewhere